store/postgres/src/chain_store.rs

```diff
@@ -1870,6 +1896,29 @@ impl ChainStore {
             .await?;
         Ok(values)
     }
+
+    pub fn rewind_chain(&self, block_ptr_to: BlockPtr) -> Result<(), StoreError> {
+        let mut conn = self.pool.get()?;
+
+        conn.transaction(|conn| {
+            use public::ethereum_networks;
+
+            self.storage
+                .delete_blocks_after(conn, &self.chain, block_ptr_to.number as i64)?;
+
+            // Update the chain head
+            diesel::update(
+                ethereum_networks::table.filter(ethereum_networks::name.eq(&self.chain)),
+            )
+            .set((
+                ethereum_networks::head_block_number.eq(block_ptr_to.number as i64),
```

Comment:
I think there is somewhere we keep the firehose cursor, I think this table has a firehose cursor too, that should be set to empty string.

Reply:
same here regarding tests